src/experiments/sonar_cnn.py

```diff
@@ -0,0 +1,88 @@
+import os
+import random
+
+import numpy as np
+from sklearn.model_selection import KFold
+
+import utils.settings as settings
+from evaluation.metrics import accuracy
+from loader.Preprocessor import Preprocessor
+from loader.load_dataset import load_dataset
+from models.JensModel import JensModel
+from training.k_fold_test import k_fold_cross_validation_test
+from utils.filter_activities import filter_activities
+from utils.folder_operations import new_saved_experiment_folder
+
+# Init
+settings.init("sonar")
+random.seed(1678978086101)
+
+print(settings.ACTIVITIES)
+# Load & Prepare data
+
+# Pass a limit to only load a few recordings (for testing)
+recordings = load_dataset("/Users/franz/Projects/BP/new_data", limit=None)
```

Comment:
maybe we could use some "data_path.json" here instead

Reply:
True, I think we even have it in settings somewhere.